packages/office-ui-fabric-react/src/components/Pivot/Pivot.styles.ts

```diff
@@ -1,14 +1,119 @@
 import { IPivotStyleProps, IPivotStyles } from './Pivot.types';
-import { normalize, FontSizes, FontWeights } from '../../Styling';
+import {
+  AnimationVariables,
+  GlobalClassNames,
+  getFocusStyle,
+  getGlobalClassNames,
+  HighContrastSelector,
+  IStyle,
+  normalize,
+  FontSizes,
+  FontWeights
+} from '../../Styling';
 
-export const getStyles = (props: IPivotStyleProps): IPivotStyles => {
-  const { className, theme } = props;
+const globalClassNames = {
+  count: 'ms-Pivot-count',
+  icon: 'ms-Pivot-icon',
+  isSelected: 'is-selected',
+  link: 'ms-Pivot-link',
+  linkContent: 'ms-Pivot-linkContent',
+  root: 'ms-Pivot',
+  rootIsLarge: 'ms-Pivot--large',
+  rootIsTabs: 'ms-Pivot--tabs',
+  text: 'ms-Pivot-text'
+};
+
+const linkStyles = (
+  props: IPivotStyleProps,
+  classNames: Partial<GlobalClassNames<typeof globalClassNames>>
+): IStyle[] => {
+  const { rootIsLarge, rootIsTabs, theme } = props;
+  const { palette } = props.theme;
+  return [
+    classNames.link,
+    {
+      color: palette.neutralPrimary,
+      display: 'inline-block',
+      fontSize: FontSizes.medium,
+      fontWeight: FontWeights.regular,
+      lineHeight: '40px',
+      marginRight: '8px',
+      padding: '0 8px',
+      textAlign: 'center',
+      position: 'relative',
+      backgroundColor: 'transparent',
+      border: 0,
+      selectors: {
+        ':before': {
+          backgroundColor: 'transparent',
+          bottom: 0,
+          content: '',
+          height: '2px',
+          left: '8px',
+          position: 'absolute',
+          right: '8px',
+          transition: `background-color ${AnimationVariables.durationValue2} ${AnimationVariables.easeFunction2}`
+        },
+        ':after': {
+          color: 'transparent',
+          // TODO: how to represent this?
+          // content: attr(title)
```

Comment:
I need to figure this out. If it's obvious or anybody knows, please let me know

Reply:
```
  content: 'attr(title)'